File: blog_springBoot/src/main/java/com/synergism/blog/core/favorite/entity/FavoriteInformation.java
package com.synergism.blog.core.favorite.entity;

import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Setter
@Getter
public class FavoriteInformation extends Favorite {

    //收藏夹下的收藏
    private List<Collection> collectionList;
    //所属用户昵称
    private String nickname;
    //所属用户账户
    private String username;
}

File: blog_vue/src/model/favorite/FavoriteInformation.ts
import Collection from "./Collection";
import Favorite from "./Favorite";

export default class FavoriteInformation extends Favorite {
  collectionList: Array<Collection>;
  nickname: string;
  username: string;

  constructor() {
    super();
    this.collectionList = [new Collection()];
    this.nickname = "";
    this.username = "";
  }
}